apps/backend/src/shared/select/job-user-view.ts

```diff
@@ -0,0 +1,39 @@
+export const JobUserViewSelect = (userId?: string) => {
```

Comment:
Can't you use `include` instead of this big select?

Reply:
No, because there is some data that cannot be returned, so if I include instead of select, I will return some data that should not be returned to the user